src/main/java/nl/fuchsia/exceptionhandlers/ErrorResponse.java

```diff
@@ -6,17 +6,17 @@
  * Deze klasse zorgt ervoor dat de errorReponse van het List formaat is.
  */
 public class ErrorResponse {
-    private List error; // TODO dit is een raw format, generic maken
+	private List error; // TODO dit is een raw format, generic maken
```

Comment:
Hier staat nog steeds een TODO.

Reply:
Er staan meerdere TODO in sinds sprint 3, ter herinnering.

Reply:
Oké, maar je het kan geen kwaad om af en toe de TODO's even langs te lopen en te fixen.

Reply:
Dat ben ik ook mee bezig geweest vanmiddag, deze kwam ik niet uit, maandag even naar kijken?